cmd/benchi/main.go

```diff
@@ -75,7 +75,7 @@ func mainE() error {
 		return fmt.Errorf("output path is required")
 	}
 
-	*outPathFlag = strings.ReplaceAll(*outPathFlag, "${now}", time.Now().Format("20060102150405"))
+	*outPathFlag = strings.ReplaceAll(*outPathFlag, "${now}", time.Now().Format("2006_01_02_15_04_05"))
```

Comment:
That's a lot of underscores 😅 How about this?
```suggestion
	*outPathFlag = strings.ReplaceAll(*outPathFlag, "${now}", time.Now().Format("20060102_150405"))
```

Reply:
Personally, I prefer the current version with more underscores, because I can more easily see the minutes and seconds. I run a few tests one after each other, and quickly looking at the hour and minutes is helpful.

Reply:
I think I'm fine either way, but I'm personally used to seeing the date like `YYYYMMDD` without dashes in between, and if what we want to be able to find easily is the hour and minutes, maybe separating the two blocks makes more sense. In other words, `day_time` (e.g.: 20250313_150303). That way, there's no confusion about which one is which. 

Either way, I'll be fine with the decision made.

Reply:
> I'm personally used to seeing the date like `YYYYMMDD`

Yeah, that's what it boils down to for me as well. I think it's more common to see it written as `date_time`.

Reply:
Let's then have it with a single underscore. :+1: